src/crypto/CMakeLists.txt

```diff
@@ -80,6 +83,7 @@ arqma_add_library(cncrypto
 target_link_libraries(cncrypto
   PUBLIC
     epee
+    randomx
```

Comment:
This should be `randomarq` otherwise it won't work as desired.

Reply:
Not really. Cus randomarq is doing randomx lib. It was tested at stagenet and working as it should

Reply:
But name interference I guess...

Reply:
RandomARQ is based on RandomX